home/lncm/.lightning/config

```diff
@@ -0,0 +1,21 @@
+network=bitcoin
+alias=LNCM BOX Default
+rgb=000000
+mainnet
+rpc-file=/root/.lightning/lightning-rpc
+
+bitcoin-rpcconnect=btcbox
+bitcoin-rpcport=8332
+bitcoin-rpcuser=RANDOMUSER
+bitcoin-rpcpassword=RANDOMPASS
+bitcoin-datadir=/home/bitcoin/.bitcoin
+bitcoin-cli=/opt/bitcoin-0.17.0/bin/bitcoin-cli
+
+log-level=debug
+log-file=/root/.lightning/debug.log
+
+proxy=10.22.0.240:9050
```

Comment:
What's this do?

Reply:
that is for communicating through the TOR network but I'll remove it because we don't need it